apps/desktop/desktop/src/main/java/bisq/desktop/main/content/settings/network/NetworkInfoView.java

```diff
@@ -54,9 +54,11 @@ public NetworkInfoView(NetworkInfoModel model, NetworkInfoController controller,
         Label myVersionAndCommitHash = new Label(Res.get("settings.network.myVersionAndCommitHash",
                 ApplicationVersion.getVersion().getVersionAsString(),
                 ApplicationVersion.getBuildCommitShortHash()));
+        Label torVersion = new Label(Res.get("settings.network.torVersion",
```

Comment:
Better add it to the `settings.network.myVersionAndCommitHash` string. As it is now, the vertical distance is different between the first 2 lines and tor version as first 2 use a linebreak in the string.

Reply:
the space was left on purpose, but we discussed on matrix and agree to try to put them together which I'm gonna do now